### DISCUSSION TITLE
Support Material You app icon theming

### DISCUSSION THREAD

**Discussion by Rarisma** [community/community#39866] [Mobile]
Hi, earlier today my phone updated to android 13 which adds a feature that dynamically themes supported app icons based on the users selected color scheme.

I'd really like it if github android could support this so that users app icons look consistent.

![Screenshot_20221123_193426_One UI Home.jpg](https://user-images.githubusercontent.com/49795711/203635238-66d770dd-b44b-4767-a84c-b6d9b7e6295b.jpg)

**Comment by taki-on** [comment]
Hi! Thank you for the suggestion! We do not support custom icon theming yet, but we already have it on our plate. Please stay tuned!

**Comment by xPaw** [comment]
This indeed would be great to have. All it needs is a monochrome icon: https://developer.android.com/develop/ui/views/launch/icon_design_adaptive

**Comment by Mysterious-Dev** [comment]
Already added for Android 13 I think.

![Screenshot_20230621-140519~2.png](https://github.com/community/community/assets/40738104/fc44dfa4-b321-4324-b896-902293ce3155)
